src/standalone.js

```diff
@@ -1,18 +1,15 @@
-import * as Babel from "@babel/core";
+import * as Babel from "@babel/standalone";
 
 // take from @babel/standalone
-import {
-  presets as availablePresets,
-  plugins as availablePlugins,
-} from "./plugins-list";
+import {availablePlugins, availablePresets} from '@babel/standalone';
 
-export function transpilePlugin(pluginString) {
+export function transpilePlugin(pluginString, presets) {
   return Babel.transform(pluginString, {
     babelrc: false,
     configFile: false,
     ast: false,
     highlightCode: false,
-    presets: [availablePresets["@babel/preset-env"]],
+    presets: presets
```

Comment:
When the property name and variable are the same, you can just supply `presets` without the value. Being explicit is good though.